frontend/src/v5/store/users/users.redux.ts

```diff
@@ -19,11 +19,14 @@ import { createActions, createReducer } from 'reduxsauce';
 import { Constants } from '../../helpers/actions.helper';
 
 export interface IUser {
-	user: string;
+	username: string;
```

Comment:
The request for members return  'user' in this field, so this generates a bug. Im guessing this is for the currentUser bit. That's in v4  (which we need to update , there is already an issue for that #3049) and it has a different shape.

Reply:
Renamed